### ISSUE TITLE
Commit EEZ Studio project files

### ISSUE THREAD

**Issue by spohtl** [meshtastic/device-ui#27]
Hi, I tried searching but couldn't find the EEZ Studio project files needed to edit or add GUI elements. Could you please commit those files as well?

**Comment by joyel24** [comment]
I would to have these files also...
Another repo or not doesn't matter but it can help.

**Comment by joyel24** [comment]
Thank you, I instantly downloaded it before it dissapear... :D
https://github.com/meshtastic/device-ui/blob/master/studio/320x240/TFT320x240.eez-project

**Comment by joyel24** [comment]
You're right, but it's a bit ironic and mostly related to the fact that I used to ask for it more than one time.
This is good news 🙏
